2022-08-29 17:14:46,222 INFO werkzeug MainThread :  * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5000
 * Running on http://10.10.10.235:5000 (Press CTRL+C to quit)
2022-08-29 17:14:49,647 INFO root Thread-1 : Get connection to ('10.100.151.128', 9669)
2022-08-29 17:14:49,660 INFO root Thread-1 : LOOKUP ON ServiceTemplateDefinition  
2022-08-29 17:14:49,661 ERROR app Thread-1 : Exception on /cluster_names [GET]
Traceback (most recent call last):
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/app/views.py", line 60, in cluster_names
    message = jsonify({'cluster_names': vid.column_values('id')})
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/json/__init__.py", line 301, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/json/__init__.py", line 131, in dumps
    return _json.dumps(obj, **kwargs)
  File "/usr/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/usr/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/dmitrytulin/PycharmProjects/API_COURSE/venv/lib/python3.9/site-packages/flask/json/__init__.py", line 50, in default
    return super().default(o)
  File "/usr/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type ValueWrapper is not JSON serializable
2022-08-29 17:14:49,663 INFO werkzeug Thread-1 : 127.0.0.1 - - [29/Aug/2022 17:14:49] "[35m[1mGET /cluster_names HTTP/1.1[0m" 500 -
